master/internal/core.go

```diff
@@ -1513,13 +1514,15 @@ func (m *Master) Run(ctx context.Context, gRPCLogInitDone chan struct{}) error {
 		return err
 	}
 
-	webhooks.Init()
-	defer webhooks.Deinit()
-	ssup := stream.NewSupervisor(m.db.URL)
-	go func() {
-		_ = ssup.Run(ctx)
-	}()
-	m.echo.GET("/stream", api.WebSocketRoute(ssup.Websocket, m.config.EnableCors))
+	if slices.Contains(m.config.FeatureSwitches, "streaming_updates") {
+		webhooks.Init()
+		defer webhooks.Deinit()
```

Comment:
talked offline but webhooks shouldn't be turned off by this feature flag.

Reply:
Thanks. I thought web socket is only used in Streaming Updates.

Reply:
websockets, webhooks and streaming updates are all different.